File: adobe-edge/src/api/details/AdobeCustomMetadataDetails.ts
/**
 * Custom metadata details information.
 *
 * https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/custommetadatadetails.schema.md
 */
export type AdobeCustomMetadataDetails = {
  // The name of the custom field.
  name?: string;

  // The value of the custom field.
  value?: string;
};

File: adobe-edge/android/src/main/java/com/theoplayer/reactnative/adobe/edge/api/AdobeCustomMetadataDetails.kt
package com.theoplayer.reactnative.adobe.edge.api

/**
 * Custom metadata details information.
 *
 * @see <a href="https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/custommetadatadetails.schema.md">Adobe XDM CustomMetadataDetails Schema</a>
 */
data class AdobeCustomMetadataDetails(
  // The name of the custom field.
  val name: String? = null,
  // The value of the custom field.
  val value: String? = null
)

